### DISCUSSION TITLE
Drop-down field in address book hidden after startup

### DISCUSSION THREAD

**Discussion by isential-gmbh** [rustdesk/rustdesk#8794] [General]
RustDesk versions 1.2.4, 1.2.6, and 1.2.7

Every time I start the application, the drop-down field in the address book is hidden. I always have to click on the hash symbol to display the drop-down field. This also happens with the latest version, 1.2.7.

![image](https://github.com/user-attachments/assets/2f0ab574-1247-42e8-8f67-efe3c386be30)

**Before closing:**
![image](https://github.com/user-attachments/assets/18f5a752-47c6-48d8-af22-cc2239fb952f)

**After startup**
![image](https://github.com/user-attachments/assets/494c2e6b-d9ab-40a7-a51b-447f5ef06cdb)

**Comment by rustdesk** [comment]
@21pages

**Comment by rustdesk** [comment]
Please create a new custom client, we had a fix for this.

**Reply by isential-gmbh** [reply]
Thank you very much. I have created a new custom version and installed it on my system: it worked perfectly right away.

Thanks again for your quick and straightforward help.

Best regards,

René

**Comment by isential-gmbh** [comment]
I am now closing the thread. Thank you!
